app/main.py

```diff
@@ -1,2 +1,40 @@
 class Dictionary:
-    pass
+    def __init__(self):
+        self.volume = 0
+        self.size = 8
+        self.table = [None] * self.size
+
+    def __setitem__(self, key, value):
+        i = hash(key) % self.size
+        while True:
+            if self.table[i] is None:
+                self.table[i] = [hash(key), key, value]
+                self.volume += 1
+                break
+            if self.table[i][1] == key:
+                self.table[i][2] = value
+                break
+            i = (i + 1) % self.size
+        if self.volume > int(self.size * 2 / 3):
+            self.resize()
+
+    def __getitem__(self, key):
+        i = hash(key) % self.size
+        for _ in range(self.size):
```

Comment:
It is better start iterate from i. Iterating every time from 0 makes dict.__getitem__ => O(n)